### PR TITLE
Fixing flake8 errors

### PR DESCRIPTION
**Pull request by nmcspadden**
Moving log and log_err to autopkglib so flake8 doesn't complain anymore.

### CODE REVIEW

File: Code/autopkg [hunk 1]
```diff
@@ -2365,7 +2353,9 @@ def run_recipes(argv):
                 # record any summary results
                 output_keys = item["Output"].keys()
                 results_keys = [
-                    key for key in output_keys if key.endswith("_summary_result")
+                    summary_key
+                    for summary_key in output_keys
+                    if key.endswith("_summary_result")
```
Comment (homebysix) [inline Code/autopkg]: Should this not be `summary_key` also?

Comment (gregneagle) [reply Code/autopkg]: Yep, looks like it to me.

Comment (gregneagle) [reply Code/autopkg]: I thought Nick was just reformatting and moving functions to other modules. Sigh

Comment (homebysix) [reply Code/autopkg]: Resolved by https://github.com/autopkg/autopkg/pull/531/files

**Pull request merged**